zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
ImagePNGRGBA8 does not make sense, since the PNG format does define the type. GL_RGBA8 vs GL_DXT3 uses the same scheme. We should discuss this offline.